userspace/libsinsp/sinsp.h

```diff
@@ -720,6 +720,34 @@ class SINSP_PUBLIC sinsp
 	}
 
 	/*!
+	  \brief If this is an online capture, set event_id.
+	  \param event type to set
+	  \return SCAP_SUCCESS if the call is succesful
+	   On Failure, SCAP_FAILURE is returned and getlasterr() can be used to
+	   obtain the cause of the error.
+
+	  \note For a list of event types, refer to \ref etypes.
+	*/
+	int32_t set_eventmask(uint32_t event_types)
+	{
+		return scap_set_eventmask(m_h, event_types);
+	}
+
+	/*!
+	  \brief If this is an online capture, unset event_id.
+	  \param event type to unset
+	  \return SCAP_SUCCESS if the call is succesful
+	   On Failure, SCAP_FAILURE is returned and getlasterr() can be used to
+	   obtain the cause of the error.
+
+	  \note For a list of event types, refer to \ref etypes.
+	*/
+	int32_t unset_eventmask(uint32_t event_id)
+	{
+		return scap_unset_eventmask(m_h, event_id);
```

Comment:
Usually we throw exceptions when scap function return error: https://github.com/draios/sysdig/blob/dev/userspace/libsinsp/sinsp.cpp#L520